src/main/scala/org/globalforestwatch/summarystats/carbonflux/CarbonFluxSummary.scala

```diff
@@ -23,15 +23,16 @@ case class CarbonFluxSummary(
 object CarbonFluxSummary {
   // CarbonFluxSummary form Raster[CarbonFluxTile] -- cell types may not be the same
 
-  implicit val mdhCellRegisterForCarbonFluxRaster1: CellVisitor[Raster[CarbonFluxTile], CarbonFluxSummary] =
-    new CellVisitor[Raster[CarbonFluxTile], CarbonFluxSummary] {
+  implicit val mdhCellRegisterForTreeLossRaster1
+    : GridVisitor[Raster[CarbonFluxTile], CarbonFluxSummary] =
+      new GridVisitor[Raster[CarbonFluxTile], CarbonFluxSummary] {
+      val acc = new CarbonFluxSummary()
 
-      def register(
-                    raster: Raster[CarbonFluxTile],
-                    col: Int,
-                    row: Int,
-                    acc: CarbonFluxSummary
-                  ): CarbonFluxSummary = {
+      def result = acc
+
+      def visit(raster: Raster[CarbonFluxTile],
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)